components/Header.tsx

```diff
@@ -32,7 +32,7 @@ const NavButtons = () => {
     { name: 'Vote', href: `/${id}/vote` },
   ];
 
-  const current = tabs.find((tab) => tab.href === pathname);
+  const current = tabs.find((tab) => pathname.includes(tab.href.split('/').at(-1) as string));
```

Comment:
What is this if not casted to a string? Is it `string | undefined?

Reply:
yep exactly; can do a logical OR here instead if you prefer